src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
```

Comment:
An `insert` function is not yet implemented as we didn't have a reason to do so. It can be very easily implemented if needed, I can add it if you think it will be needed.